src/coordsim/decision_maker/default_decision_maker.py

```diff
@@ -0,0 +1,71 @@
+import logging
+import numpy as np
+from coordsim.network.flow import Flow
+from coordsim.simulation.simulatorparams import SimulatorParams
+from coordsim.decision_maker import BaseDecisionMaker
+log = logging.getLogger(__name__)
+
+
+class DefaultDecisionMaker(BaseDecisionMaker):
+    """
+    This is the default decision maker class. It makes flow decisions based on the scheduling table
+    """
+
+    def __init__(self, env, params):
+        self.env = env
```

Comment:
Why not do that in the Base class?

Reply:
Then we would have to call `super()` in each class. Not a problem, I can switch to using that.

Reply:
You wouldn't even need to implement `__ini__()` in the child classes if they don't need any special setup.

Reply:
They would simply inherit the constructor of the parent. So it would reduce duplicate code.